doc/ref_arch/openstack/chapters/chapter08.rst

```diff
@@ -6,7 +6,7 @@ to validate OpenStack based cloud infrastructure
 against the standard set of requirements defined in
 :ref:`chapters/chapter02:Architecture Requirements`. Through this validation
 mechanism, a provider of cloud infrastructure will be able
-to test their conformance to this reference architecture. This will ease the
+to test the conformance to this reference architecture. This will ease the
```

Comment:
to be exact, we should change to either of the following:

"to test the conformance of their cloud infrastructure
to this reference architecture. This will ease the"

or

"to test their cloud infrastructure's conformance
to this reference architecture. This will ease the"

Reply:
@pgoyal01 Suggestion included